Packs/GCP/Integrations/GCP/GCP.yml

```
@@ -141,7 +141,441 @@ script:
+    - name: source_ranges
+      description: Comma-separated CIDRs for INGRESS.
+    - name: destination_ranges
+      description: Comma-separated CIDRs for EGRESS.
+    - name: source_tags
+      description: Comma-separated instance tags to match as source.
+    - name: target_tags
+      description: Comma-separated tags to apply this rule to.
+    - name: source_service_accounts
+      description: Comma-separated service accounts for source.
+    - name: target_service_accounts
+      description: Comma-separated service accounts to target.
+    - name: log_config_enable
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Enable firewall logging.
+    - name: disabled
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Whether this firewall rule is disabled.
+    outputs:
+    - contextPath: GCP.Compute.Operations.id
+      description: Unique identifier for the operation resource, defined by the server.
+      type: String
+    - contextPath: GCP.Compute.Operations.name
+      description: Name of the operation resource.
+      type: String
+    - contextPath: GCP.Compute.Operations.kind
+      description: Type of the resource, for example compute#operation.
+      type: String
+    - contextPath: GCP.Compute.Operations.operationType
+      description: Type of operation, such as insert, update, or delete.
+      type: String
+    - contextPath: GCP.Compute.Operations.status
+      description: Current status of the operation.
+      type: String
+    - contextPath: GCP.Compute.Operations.progress
+      description: Progress of the operation as a percentage between 0 and 100.
+      type: Number
+    - contextPath: GCP.Compute.Operations.targetId
+      description: Unique target ID of the resource affected by the operation.
+      type: String
+    - contextPath: GCP.Compute.Operations.targetLink
+      description: URL of the target resource modified by the operation.
+      type: String
+    - contextPath: GCP.Compute.Operations.selfLink
+      description: Server-defined URL for the operation resource.
+      type: String
+    - contextPath: GCP.Compute.Operations.insertTime
+      description: The time when the operation resource was created.
+      type: Date
+    - contextPath: GCP.Compute.Operations.startTime
+      description: The time when the operation started running.
+      type: Date
+    - contextPath: GCP.Compute.Operations.user
+      description: The user account that performed the operation.
+      type: String
+  - name: gcp-compute-firewall-list
+    description: Lists firewall rules in the specified project.
+    arguments:
+    - name: project_id
+      required: true
+      description: GCP project ID.
+    - name: limit
+      defaultValue: '500'
+      description: "Maximum number of results to return. Acceptable values are 0 to 500, inclusive. (Default: 500)."
```

```suggestion
      description: "Maximum number of results to return. Acceptable values are 0 to 500, inclusive."
```
as we discussed, the defaultValue will be shown in the UI.